extras/indexingExample/src/main/java/ProspectorExample.java

```diff
@@ -0,0 +1,193 @@
+
+    private static final URI WORKS_AT = VALUE_FACTORY.createURI("urn:worksAt");
+    private static final URI ADMIRES = VALUE_FACTORY.createURI("urn:admires");
+    private static final URI LIVES_WITH = VALUE_FACTORY.createURI("urn:livesWith");
+
+    private static final URI BURGER_JOINT = VALUE_FACTORY.createURI("urn:burgerJoint");
+    private static final URI DONUT_SHOP= VALUE_FACTORY.createURI("urn:donutShop");
+
+    public static void main(final String[] args) throws Exception {
+        setupLogging();
+
+        // Configure Rya to use a mock instance.
+        final AccumuloRdfConfiguration config = new AccumuloRdfConfiguration();
+        config.useMockInstance(true);
+        config.setTablePrefix("rya_");
+        config.setUsername("user");
+        config.setPassword("pass");
+        config.setInstanceName("accumulo");
+
+        // Load some data into Rya.
+        final List<Statement> statements = Lists.newArrayList(
+                VALUE_FACTORY.createStatement(ALICE, WORKS_AT, BURGER_JOINT),
+                VALUE_FACTORY.createStatement(ALICE, ADMIRES, BOB),
+                VALUE_FACTORY.createStatement(BOB, WORKS_AT, DONUT_SHOP),
+                VALUE_FACTORY.createStatement(CHARLIE, WORKS_AT, DONUT_SHOP),
+                VALUE_FACTORY.createStatement(CHARLIE, LIVES_WITH, BOB),
+                VALUE_FACTORY.createStatement(BOB, LIVES_WITH, CHARLIE),
+                VALUE_FACTORY.createStatement(BOB, LIVES_WITH, ALICE));
+
+        final Sail sail = RyaSailFactory.getInstance(config);
+        final SailConnection conn = sail.getConnection();
+        log.info("Loading the following statements into a Mock instance of Accumulo Rya:");
+        conn.begin();
+        for(final Statement statement : statements) {
+            log.info("    " + statement.toString());
+            conn.addStatement(statement.getSubject(), statement.getPredicate(), statement.getObject());
+        }
+        conn.commit();
+        conn.close();
+
+        // Create the table that the Prospector's results will be written to.
+        ConnectorFactory.connect(config)
+                .tableOperations()
+                .create("rya_prospects");
+
+        // Run the Prospector using the configuration file that is in the resources directory.
+        log.info("");
+        log.info("Running the Map Reduce job that computes the Prospector results.");
+        ToolRunner.run(new Prospector(), new String[]{ "src/main/resources/stats_cluster_config.xml" });
+
+        // Print the table that was created by the Prospector.
+        log.info("");
+        log.info("The following cardinalities were written to the Prospector table:");
+        final ProspectorServiceEvalStatsDAO dao = ProspectorServiceEvalStatsDAO.make(config);
+
+        // Do each of the Subjects.
+        double cardinality = dao.getCardinality(config, CARDINALITY_OF.SUBJECT, Lists.newArrayList(ALICE));
```

Comment:
I think reducing the lines of the code, in this case, actually makes the example a lot harder to understand.